src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -22,22 +26,22 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 Optional.of(new Address("Blk 30 Geylang Street 29, #06-40")),
-                getTagSet("friends")),
+                getTagSet("friends"), getMap("CS1231S", RoleType.STUDENT)),
             new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                 Optional.of(new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18")),
-                getTagSet("colleagues", "friends")),
+                getTagSet("colleagues", "friends"), getEmptyMap()),
             new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
                 Optional.of(new Address("Blk 11 Ang Mo Kio Street 74, #11-04")),
-                getTagSet("neighbours")),
+                getTagSet("neighbours"), getMultipleRolesMap()),
             new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
                 Optional.of(new Address("Blk 436 Serangoon Gardens Street 26, #16-43")),
-                getTagSet("family")),
+                getTagSet("family"), getEmptyMap()),
             new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
                 Optional.of(new Address("Blk 47 Tampines Street 20, #17-35")),
-                getTagSet("classmates")),
+                getTagSet("classmates"), getMap("CS1522", RoleType.TUTOR)),
```

Comment:
not sure if intended, but maybe we should use real module codes since we can choose to

Reply:
Thank you for pointing it out! It's another typo